docs/Secure-Coding-Guide-for-Python/CWE-703/CWE-755/noncompliant01.py

```diff
@@ -0,0 +1,18 @@
+""" Non-compliant Code Example """

```

Comment:
Add license comments on top of all the .py files. You don't need to put them in code chunks in readme files:
`# SPDX-FileCopyrightText: OpenSSF project contributors`
`# SPDX-License-Identifier: MIT`